rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```diff
@@ -359,6 +359,45 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
     setvar:'tx.inbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
 
 
+# [ Unix shell expressions - Bash Tilde expansion ]
+#
+# Detects the following patterns which are common in Unix shell scripts
```

Comment:
This comment should also be part of the assembly file (slightly modified for the context).